engine/render/renderer/renderer/BillboardRenderer.cpp

```diff
@@ -84,7 +84,7 @@ void BillboardRenderer::DrawBillboard(const Vec3& position,
 {
     auto targetTexture = texture == nullptr ? &defaultTexture : texture;
```

Comment:
@jonjondev  - I know it looks like it, but we've actually reduced the number of derefs. Beforehand we'd pass the pointer to the function and it would dereference the pointer (using ->) multiple times. Now we only dereference it once.